metamist/parser/generic_parser.py

```diff
@@ -682,31 +728,33 @@ def prepare_message(
 
         assays_count: dict[str, int] = defaultdict(int)
         sequencing_group_counts: dict[str, int] = defaultdict(int)
-        for s in assays:
-            assays_count[str(s.assay_type)] += 1
+        for a in assays:
+            assays_count[str(a.meta.get('sequencing_type'))] += 1
```

Comment:
Can you we do this by both the assay_type and sequencing_type, even though the former will just be 'sequencing' for now.

Reply:
Added the count of `assay.assay_type` back in in addition to the `assay.meta.sequencing_type` count.